src/components/InfoPanel/InfoPanel.test.js

```diff
@@ -4,8 +4,25 @@ import { shallow } from "enzyme";
 import InfoPanel from "./InfoPanel";
 
 describe("Info Panel", () => {
+  const modelStatus = {
```

Comment:
After making the above change to use the `selector` you can use the real 'mock' data which will allow us to keep our tests more robust as they are all using the same data set.